doc/psud/PSU_daemon_design.md

```diff
@@ -0,0 +1,152 @@
+# SONiC PSU Daemon Design #
+
+### Rev 0.1 ###
+
+### Revision ###
+
+ | Rev |     Date    |       Author       | Change Description                |
+ |:---:|:-----------:|:------------------:|-----------------------------------|
+ | 0.1 |             |      Chen Junchao  | Initial version                   |
+
+
+## 1. Overview
+
+When I write this document, SONiC already has a PSU daemon called psud running in pmon docker. There is already a high level design document which contains existing psud [design](https://github.com/Azure/SONiC/blob/master/doc/pmon/pmon-enhancement-design.md), but some of its content is out of date. This document will update the design of current PSU daemon implementation and add an enhancement about PSU led management.
```

Comment:
this is relevant to the design doc, should be described in the PR description not here.